examples/dtls-wolfssl/dtls-server.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/*
+ * **** This file incorporates work covered by the following copyright and ****
+ * **** permission notice:                                                 ****
+ *
+ * Copyright (C) 2006-2018 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ *
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Demonstrating DTLS 1.2 client using wolfSSL
+ *
+ * @author      Daniele Lacamera <daniele@wolfssl.com>
+ * @}
+ */
+
+#include <wolfssl/ssl.h>
+#include <sock_tls.h>
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include <net/sock/udp.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define SERVER_PORT 11111
+#define DEBUG 1
+extern const unsigned char server_cert[788];
+extern const unsigned char server_key[121];
+extern unsigned int server_cert_len;
+extern unsigned int server_key_len;
+
+static sock_tls_t skv;
+static sock_tls_t *sk = &skv;
```

Comment:
minor: why the additional pointer?

Reply:
just for readability, to avoid extra-dereferences in the code below.